### ISSUE TITLE
limb length

### ISSUE THREAD

**Issue by hyli2020** [tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation#89]

**Comment by tensorboy** [comment]
Yes, the limb_width is 1 in default, I tried to set it to more than 1 (1.1, 1.2, 1.3), it shows a slight increase in the mAP.

**Comment by hyli2020** [comment]
> Yes, the limb_width is 1 in default, I tried to set it to more than 1 (1.1, 1.2, 1.3), it shows a slight increase in the mAP.

is that so? I saw the limb_length=1.2 here.  https://github.com/tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation/blob/master/train_SH.py#L77

**Comment by hyli2020** [comment]
One more thing, why keypoint (2-16) is defined as limb?https://github.com/tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation/blob/master/training/datasets/coco_data/COCO_data_pipeline.py#L186-#L190

Also refer to the visualization here:
https://github.com/CMU-Perceptual-Computing-Lab/openpose/blob/master/doc/output.md#pose-output-format-coco

**Comment by tensorboy** [comment]
You mean why keypoint connected in this way?

The limb is the line between the corresponding key point, it can be defined in other ways actually.

**Comment by hyli2020** [comment]
I know. I mean why the shoulder (id 2) and ear (id 16) are connected? Please refer to the visualization link above.

**Comment by tensorboy** [comment]
I see. The connection between these two points did not influence the final result as far as I know, and it can be safely ignored in the processing to handle PAF. The pipeline is mainly to respect the original C++ version and converted from here.

**Comment by tensorboy** [comment]
And if you want to improve the mAP further, the accuracy of key points is essential as I know. Illustrated in 'http://openaccess.thecvf.com/content_ECCV_2018/papers/Bin_Xiao_Simple_Baselines_for_ECCV_2018_paper.pdf':"it seems that obtaining high resolution feature maps is crucial, but no matter how."

And [mask rcnn](https://arxiv.org/pdf/1703.06870.pdf): "We found that a relatively high resolution output (compared to masks) is required for keypoint-level localization accuracy. "

Or the newest CVPR work: https://github.com/leoxiaobin/deep-high-resolution-net.pytorch mainly to improve the resolution. :)

**Comment by tensorboy** [comment]
One more thing, are you interested in to join AiFi (http://nanostore.ai/) in Santa Clara? We just finished the Series A round several days ago.
